### PR TITLE
fix(chat): fix broken chat settings UI for mobile view (Issue #1653)

### PR DESCRIPTION
**Pull request by denys-kolomiitsev**
**Description:**

- Fix the broken UI for the chat settings modal in mobile view

Issues:

- Issue #1653 

**UI changes**

image

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1659.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Sidebar/Sidebar.tsx [hunk 1]
```diff
@@ -86,6 +87,10 @@ const Sidebar = <T,>({
 
   const isOverlay = useAppSelector(SettingsSelectors.selectIsOverlay);
 
+  const [windowWidth, setWindowWidth] = useState(
```
Comment (Alexander-Kezik) [inline apps/chat/src/components/Sidebar/Sidebar.tsx:90]: here type will be `useState<number | false>` is it expected?

File: apps/chat/src/components/Sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -86,6 +87,10 @@ const Sidebar = <T,>({
...
+    typeof window !== 'undefined' && window.innerWidth,
+  );
```
Comment (Alexander-Kezik) [inline apps/chat/src/components/Sidebar/Sidebar.tsx:92]: here type will be `useState<number | false>` is it expected?

maybe it's better to write like `typeof window !== 'undefined' ? window.innerWidth : undefined`

**Review by Alexander-Kezik — APPROVED** [review]

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1659.nightly-test.deltixhub.io
>E2E tests status: success

**Pull request merged**